specs/TwoPhase_anim.tla

```diff
@@ -57,7 +23,7 @@ RMElems == [i \in RMIdDomain |-> Circle(RMSpacing * i, 45, 10,
             ELSE IF rmState[RMId[i]] = "committed" THEN CommitColor 
             ELSE IF rmState[RMId[i]] = "aborted" THEN AbortColor ELSE "gray"])]
 
-TMXpos == RMSpacing * (Cardinality(RM) + 1) \div 2
+TMXpos == RMSpacing * ((Cardinality(RM) + 1) \div 2)
```

Comment:
@will62794 TLC rejects the formula without the parentheses, complaining about the precedence conflict between the division (\) and multiplication (*) operators.